### DISCUSSION TITLE
How to install torchio if the version of torch is 1.0?

### DISCUSSION THREAD

**Discussion by buaaduke** [TorchIO-project/torchio#496] [Q&A] [answered]
I want to use torchio for data augmentation. When I use the command 'pip install torchio,  it seems to upgrade my torch.  How to install torchio without impacting the current version of torch such as 1.0？

**Comment by fepegar** [comment, marked as answer]
I guess you can try [ignoring the dependencies](https://stackoverflow.com/questions/12759761/python-pip-force-install-ignoring-dependencies). But I can't guarantee everything is going to work. PyTorch 1.0 is from 2018. I suggest you update your code if it's not compatible with newer PyTorch versions.

**Reply by buaaduke** [reply]
Thank you very much. I'll have a try.
